### DISCUSSION TITLE
WGSL: How can I get the minimum float value?

### DISCUSSION THREAD

**Discussion by javirk** [gfx-rs/wgpu#3260] [General]
In GLSL I would use `uintBitsToFloat(0x2F800000u)` to get the minimum f32 (following [this](https://www.shadertoy.com/view/Mt3cRX)). However, that function doesn't seem to exist in WGSL. Is there any workaround for that?

**Comment by cwfitzgerald** [comment]
`bitcast<f32>(0x2F800000u)` should do what you want

**Reply by javirk** [reply]
Yes! That's what I wanted, thank you!
